lib/version.c

```diff
@@ -339,6 +339,21 @@ static const char * const protocols[] = {
 #ifdef USE_LIBRTMP
   "rtmp",
 #endif
+#ifdef USE_LIBRTMP
+  "rtmpe",
+#endif
+#ifdef USE_LIBRTMP
+  "rtmps",
+#endif
+#ifdef USE_LIBRTMP
+  "rtmpt",
+#endif
+#ifdef USE_LIBRTMP
+  "rtmpte",
+#endif
+#ifdef USE_LIBRTMP
+  "rtmpts",
+#endif
```

Comment:
I think you could put them all within a single `#ifdef` clause, it would look less weird to me.

Reply:
Agreed, but I wanted to respect the comment above.
Will change.

Reply:
Change pushed.